### DISCUSSION TITLE
Pinball machines

### DISCUSSION THREAD

**Discussion by curif** [curif/AgeOfJoy-2022.1#71] [Ideas]
The idea of include pinball machines in the gallery is something that I been thinking for a while. The simulation looks incomplete without pinball machines, no classic gallery would be complete without them.

AGE of Joy videogames are supported by `libretro-mame32-plus`, the implementation was not easy, and the development to display the games in the CRTs TV was hard too. But pinball machines are hardest, because you have to emulate some thing that is "physical" and mechanical, you have relays, and actuators, lights, etc.

The way is obviously libPinMAME. https://github.com/vpinball/pinmame/blob/master/src/libpinmame/libpinmame.h

It is not possible to include something pre-made like [Visual Pinball Engine](https://github.com/freezy/VisualPinball.Engine), Quest can't handle something like that.

**Comment by WilsonBradley** [comment]
Are you talking about a mock-up (fake pinball machine?) or actual within the AgeOfJoy on Quest? 
If actual, it seems that is far to demanding and why "reinvent the wheel?"
Maybe have a door that says Pinballs , when you go into it, it launches a separate Quest Pinball game that are already created.
Seems this would be far out on the roadmap, consider all the other things that need attention.

Maybe in meantime, place a static 3-D render of a Pinball machine.
https://3dmdb.com/en/3d-models/pinball/

**Comment by sassacontrol** [comment]
Existe o Pimbal FX VR para o Quest 3, Seria possível desmontar esse jogo e usar as Pimbal's dele?
